src/runner/runner.go

```diff
@@ -240,3 +242,8 @@ func dumpMetrics(clientID string) error {
 
 	return nil
 }
+
+func resetMetrics(clientId string) {
+	metrics.Default.Write(metrics.Traffic, clientId, 0)
```

Comment:
I can't remember whether this should be a client id or job id for it to work properly, did you test this? (I'll take a look in an hour or so myself, not near my PC yet)

Reply:
Yeah, I think you need to reset the tracker as a whole (lines 53-54). I think it's best to define that method on a storage type, trying to think whether it should be protected by some lock or not

Reply:
Yep, I tested it with clientID, will move it to Storage, thank you.

Reply:
I made the reset logic a method of Storage and reset them as a whole. Since metricsTracker uses concurrent sync.Map, I assume there’s no need to handle synchronization manually. Am I right?
Thinking further as a separate MR: that’s a good idea to make a separate metrics for targets. Maybe it will make sense to give a list of targets with their metrics bellow the aggregated values? So when a config is reset it’ll be pretty obvious for the user to recognise the meaning of the updated data.

Reply:
I'm almost sure that it should be safe to just replace a pointer but lets see if any linters object to that
As for metrics per target - be aware that we have support for encrypted targets and they shouldn't be shown to the user for security reasons

Reply:
The only thing linter was complaining: line length in prometheus.go (fixed).

I see, considering encrypted targets, it will raise questions if we screen them in the output. If we omit them - aggregated output will become inconsistent with discrete metrics. Leaving it as is for now seems like the best choice